apple/standalone.py

```diff
@@ -29,18 +28,18 @@ def standalone(file):
         file_new = os.path.join(librarypath, os.path.basename(file))
         if not os.path.isfile(file_new):
             copyfile(file, file_new)
-        print("Dependency {} installed.".format(os.path.basename(file)))
+        print(f"Dependency {os.path.basename(file)} installed.")
```

Comment:
f-strings need at least Python 3.6. Is this version requirement already encoded somewhere?

Reply:
We can add it to e.g. the setup.py. Maybe, we can also use a more recent version than Python 3.6 as the minimum version. What do you think?

Reply:
In reality we do not build for any Python older than 3.8 I believe. You can add this to `setup.py` (untested):
```python
setup(
    name = "pairinteraction",
    python_requires = ">3.8",
```